src/GovUk.Education.ExploreEducationStatistics.Content.Model/UserPublicationInvite.cs

```diff
@@ -0,0 +1,35 @@
+#nullable enable
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model
+{
+    public class UserPublicationInvite
```

Comment:
We've done work to manage these in the BAU pages, but it doesn't seem like there's anything to handle the conversion of them to user publication roles when the user is created?

Are we planning on doing this in a separate ticket?

Reply:
You mean the creation of the `UserPublicationRole`s from the `UserPublicationInvites` when a newly invited user first logs in? That code is in `ExternalLogin.cshtml.cs` in method `HandleNewInvitedUser`.